src/message.ts

```diff
@@ -29,6 +29,7 @@ type RequestKind =
   | "keepAlive"
   | "heartbeat";
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 type ResponseKind =
```

Comment:
Though this is not used right now, it helps reader understand the event types so we should keep it.